ddtrace/contrib/internal/azure_eventhubs/patch.py

```diff
@@ -0,0 +1,219 @@
+from typing import Dict
+
+import azure.eventhub as azure_eventhub
+import azure.eventhub.aio as azure_eventhub_aio
+from wrapt import wrap_function_wrapper as _w
+
+from ddtrace import config
+from ddtrace.contrib.internal.trace_utils import unwrap as _u
+from ddtrace.ext import azure_eventhubs as azure_eventhubsx
+from ddtrace.internal.schema import schematize_service_name
+from ddtrace.internal.utils import get_argument_value
+from ddtrace.internal.utils.formats import asbool
+from ddtrace.settings._config import _get_config
+from ddtrace.trace import Pin
```

Comment:
```suggestion
from ddtrace._trace.pin import Pin
```

`ddtrace.trace.Pin` is deprecated

Reply:
which, actually, thanks... we have a CI lint check for this, but apparently it isn't running 😅